packages/api-catalog-service/src/namespace/typedef.graphql

```diff
@@ -68,12 +68,21 @@ input NamespaceInput {
 type Query {
   listNamespaces: [NamespaceType]
   getNamespaceById(_id: ID!): NamespaceType
+  fetchAPISchema(category: ApiCategory, environment:NSEnvironmentInput): String
 }
 
 type Mutation {
   createNamespace(payload: NamespaceInput!): NamespaceType
-  updateNamespace(_id:ID!, payload: NamespaceInput!): NamespaceType
```

Comment:
Can we change the argument name to id instead of _id. It feels a bit unconventional

Reply:
Will be changed in next PR